finance/community-updates/2020-08.txt

```diff
@@ -35,7 +35,7 @@ with some updates on our work, and reminders about ongoing activities.
   and able to accept funding for this year.   We were then able to offer
   support of up to $11,000 to the following contributors: Kelle Cruz,
   Adrian Price-Whelan, Lia Corrales, Tom Robitaille, Stuart Mumford, Ole
-  Streicher, Derek Homer, Stuart Littlefair, Juan Luis Cano Rodrigues,
+  Streicher, Derek Homeier, Stuart Littlefair, Juan Luis Cano Rodrigues,
```

Comment:
Miguel de Val-Barro should be removed from this list.

Reply:
This is text that was sent out a few months ago. I think it was worth fixing the typo for posterity, but I don't think we want to rewrite the past. :-)

Reply:
Gotcha.  I missed this was the old email!